proxy/handler.go

```diff
@@ -60,6 +60,9 @@ func NewProxyHandler(logger *logging.Logger, config *config.Configuration, metri
 	}
 }
 
+
+// avoid func `(*ProxyHandler).replaceBackendUri` is unused (unused)
+// nolint: unused
 func (p *ProxyHandler) replaceBackendUri(value string, req *http.Request, appCfg *config.Application) string {
```

Comment:
So -- do we still need this function, if it's unused?

Reply:
No, we dont need it i think